CMakeLists.txt

```diff
@@ -98,6 +98,7 @@ option(ENABLE_CheMPS2 "Enables CheMPS2 for DMRG (requires HDF5)" OFF)
 option(ENABLE_cppe "Enables CPPE for Polarizable Embedding" OFF)
 option(ENABLE_adcc "Enables adcc for algebraic-diagrammatic construction methods (can also be added at runtime)" OFF)
 option(ENABLE_dkh "Enables DKH integrals (requires Fortran)" OFF)
+option(ENABLE_ecpint "Enables ECP integrals" OFF)
```

Comment:
This could be more descriptive. I assume once we completely switch to ecpint, it'll be required for any ECP calculation, and the user might not know what "ECP integrals" are.